lib/SILOptimizer/Mandatory/DefiniteInitialization.cpp

```diff
@@ -785,6 +785,7 @@ void LifetimeChecker::doIt() {
       break;
     case DIUseKind::LoadForTypeOfSelf:
       handleLoadForTypeOfSelfUse(Use);
+      break;
```

Comment:
I don't this actually effected anything since this is the last entry in the switch, but I saw this so I just wanted to handle it quickly just to prevent future bugs.